semantics/src/main/scala/aqua/semantics/expr/ConstantSem.scala

```diff
@@ -27,12 +27,12 @@ class ConstantSem[S[_]](val expr: ConstantExpr[S]) extends AnyVal {
             case true =>
               Raw.empty(s"Constant with name ${expr.name} was already defined, skipping")
             case false =>
-              Raw.error(s"Constant with name ${expr.name} was defined with different type")
+              ErroredPart(expr.name.value)
           }
         case (Some(_), _, _) =>
           Raw.error(s"Name '${expr.name.value}' was already defined").pure[Alg]
```

Comment:
Why here it is not `ErroredPart`?

Reply:
because this constant will be already in parts